internal/output/plain.go

```diff
@@ -101,13 +101,13 @@ func (f *Plain) write(writer io.Writer, value interface{}) {
 
 // writeNow is a little helper that just writes the given value to the requested writer (no marshalling)
 func (f *Plain) writeNow(writer io.Writer, value string) {
-	_, err := colorize.Colorize(wordWrap(value), writer, !f.cfg.Colored)
+	_, err := colorize.Colorize(WordWrap(value), writer, !f.cfg.Colored)
 	if err != nil {
 		logging.ErrorNoStacktrace("Writing colored output failed: %v", err)
 	}
 }
 
-func wordWrap(text string) string {
+func WordWrap(text string) string {
```

Comment:
I need to expose this for a unit test that fails on CI due to a smaller terminal width than I have locally. (My local test passes.)